package.json

```diff
@@ -62,9 +62,14 @@
     "@sapphire/framework": "2.0.0-next.eebecfb6.0",
     "@sapphire/type": "^2.0.11",
     "@sapphire/utilities": "^2.0.1",
-    "@tmware/status-rotate": "^1.2.6",
+    "@tmware/status-rotate": "^1.2.8",
     "app-root-path": "^3.0.0",
     "convict": "^6.2.0",
     "discord.js": "^12.5.3"
+  },
+  "optionalDependencies": {
```

Comment:
these are optional for discord.js, but should be added as dependencies to your bot